backend/services.plr-intake/Data/PlrDbContext.cs

```diff
@@ -40,10 +40,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
     private void ApplyAudits()
     {
-        this.ChangeTracker.DetectChanges();
-        var updated = this.ChangeTracker.Entries()
-            .Where(x => x.Entity is BaseAuditable
-                && (x.State == EntityState.Added || x.State == EntityState.Modified));
+        var updated = this.ChangeTracker.Entries<BaseAuditable>()
+            .Where(x => x.State is EntityState.Added or EntityState.Modified);
```

Comment:
This works??!? What's the "or"? I mean it is obvious what it is supposed to do, but this syntax is new to me...

Reply:
This is the pattern matching syntax added in C# 9:
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/patterns#logical-patterns

Looks crazy the first time you see it :P.